sormas-ui/src/main/java/de/symeda/sormas/ui/utils/components/sidecomponent/SideComponentField.java

```diff
@@ -27,4 +32,13 @@ public SideComponentField() {
 	public void addComponentToField(Component component) {
 		mainLayout.addComponent(component);
 	}
+
+	public void adEditButton(String id, Button.ClickListener editClickListener) {
```

Comment:
Typo: `addEditButton`

Reply:
Additionally, this method moved here from `VaccinationList`. It is also implemented in the following classes derived from `SideComponentField`:
- `TravelEntryListEntry`
- `CaseListEntry`
- `ImmunizationListEntry`
- `ContactListEntry`
The respected methods `addEditListener` can be omitted and `SideComponentField.addEditButton` used instead.